backend/server/index.js

```diff
@@ -0,0 +1,26 @@
+const express = require('express');
+const fs = require('file-system');
+const http = require('http');
+const admin = require('firebase-admin');
+
+const HelloService = require('./services/HelloService');
+
+const app = express();
+
+/* For when HTTPS is implemented
+const privKey = fs.readFileSync('./security/comakeKey.pem', 'utf8');
+const privKeyPassphrase = fs.readFileSync('./security/comakeKeyPassphrase.txt', 'utf8');
+const certificate = fs.readFileSync('./security/comakeCert.pem', 'utf8');
+const creds = { key: privKey, passphrase: privKeyPassphrase, cert: certificate };
+*/
+
+admin.initializeApp({
```

Comment:
I don't think we need to do this since the function definition would be in the same file and the functions would only ever be called once, I'll add comments for the sections though to clarify